data/digi-technology.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Digi Technology Nepal",
+    "establishedAt": 757361700,
+    "location": "Dillibazar, Kathmandu",
+    "website": "https://digitechnologynepal.com/",
+    "email": "info@digitechnologynepal.com",
+    "phone": "+977 9801986455",
```

```suggestion
    "phone": "+977-9801986455",
```
Also do update the number if there is a landline number available.